lib/landrush/action/common.rb

```diff
@@ -71,7 +71,7 @@ def read_machine_hostname
           return machine.config.vm.hostname
         end
 
-        "#{Pathname.pwd.basename}.#{config.tld}"
+        "#{Pathname.pwd.basename}.#{config.tld[0]}"
```

Comment:
So why the first element of the array? Don't you have to take any action for potential others? This looks a bit fishy to me and makes me wonder whether what one really wants is a config option which lets you define host names to be made visible on the host. There is _config.landrush.host_, could that not be used? One would just make sure that one calls the _ResolverConfig_ for each configured host. WDYT?